### PR TITLE
Add instruction to update .bash_profile

### PR DESCRIPTION
**Pull request by AntonKarabaza**
Modifications of .bash_profile during installing and uninstalling were removed.
Now there are suggestions how to update .bash_profile.

Affects #21

### CODE REVIEW

File: install.bash [hunk 1]
```diff
@@ -29,22 +29,16 @@ install() {
...
 
-    echo "# allow pem command" >> ~/.bash_profile
-    echo "export PATH=$PEM_INSTALL_PATH/bin"':$PATH' >> ~/.bash_profile
-    echo "# allow pem completion" >> ~/.bash_profile
-    echo "[ -f $PEM_INSTALL_PATH/bin/pem-completion ] && . $PEM_INSTALL_PATH/bin/pem-completion" >> ~/.bash_profile
-    echo ============== ~/.bash_profile ================
-    grep pem ~/.bash_profile
-    echo ===============================================
+    echo "To allow pem command and pem completion add the following to your .bash_profile:"
+    echo "  export PATH=$PEM_INSTALL_PATH/bin"':$PATH'
```
Comment (extsoft) [inline install.bash]: @AntonKarabaza Pass whole message into "". Remove spare single quotes.

File: install.bash [hunk 2]
```diff
@@ -29,22 +29,16 @@ install() {
...
     fi
 
-    echo "# allow pem command" >> ~/.bash_profile
-    echo "export PATH=$PEM_INSTALL_PATH/bin"':$PATH' >> ~/.bash_profile
-    echo "# allow pem completion" >> ~/.bash_profile
-    echo "[ -f $PEM_INSTALL_PATH/bin/pem-completion ] && . $PEM_INSTALL_PATH/bin/pem-completion" >> ~/.bash_profile
-    echo ============== ~/.bash_profile ================
-    grep pem ~/.bash_profile
-    echo ===============================================
+    echo "To allow pem command and pem completion add the following to your .bash_profile:"
```
Comment (extsoft) [inline install.bash]: @AntonKarabaza rewrite sentence in more easier format.

**Review by extsoft — CHANGES_REQUESTED** [review]
Also please review your commit message (at least you missed # before issue number).
Please ask me to review next time with "@extsoft" mention.

File: install.bash (hunk 1 above)
Comment (AntonKarabaza) [reply install.bash]: But it will expand the $PATH variable to output.

Comment (extsoft) [reply install.bash]: > But it will expand the $PATH variable to output.

@AntonKarabaza What about escaping? `echo "\$PATH"`

Comment (AntonKarabaza) [reply install.bash]: > What about escaping? `echo "\$PATH"`

@extsoft Missed this, thank you.

File: install.bash [hunk 3]
```diff
@@ -29,22 +29,18 @@ install() {
...
-    echo "# allow pem completion" >> ~/.bash_profile
-    echo "[ -f $PEM_INSTALL_PATH/bin/pem-completion ] && . $PEM_INSTALL_PATH/bin/pem-completion" >> ~/.bash_profile
-    echo ============== ~/.bash_profile ================
-    grep pem ~/.bash_profile
-    echo ===============================================
+    echo "Add the following to your .bash_profile to allow:"
+    echo "  pem command"
+    echo "    export PATH=$PEM_INSTALL_PATH/bin:\$PATH"
+    echo "  pem completion"
+    echo "    [ -f $PEM_INSTALL_PATH/bin/pem-completion ] && . $PEM_INSTALL_PATH/bin/pem-completion"
```
Comment (extsoft) [inline install.bash]: @AntonKarabaza, the best way to add this instructions is to copy and paste. But in current implementation I can't do that:
1. Please add # before comments
2. Please make it in copy/paste format (remove extra spaces in the begging go the line

**Review by extsoft — CHANGES_REQUESTED** [review]

**Comment by AntonKarabaza** [comment]
@extsoft Pull request closed due to change of head branch.

**Pull request closed without merging**